service/experiment-requests.yaml

```diff
@@ -20,6 +20,12 @@
 # Please add new experiment requests towards the top of this file.
 #
 
+- experiment: 2022-10-04-um
+  description: "UM fuzzer experiment"
+  fuzzers:
+    - aflplusplus
+    - aflplusplus_um_random
```

Comment:
Just request all of them here. Let's debug as much as possible, the resources aren't as important.

Reply:
oh, I only fixed one of the fuzzers currently, since everything else is just basically copy paste of this with one minor change, everything should work if this works.